thefuck/rules/rm_root.py

```diff
@@ -6,11 +8,19 @@
 @sudo_support
 def match(command):
     return (command.script_parts
-            and {'rm', '/'}.issubset(command.script_parts)
-            and '--no-preserve-root' not in command.script
-            and '--no-preserve-root' in command.output)
+            and {'rm', '/'}.issubset(command.script_parts))
 
 
-@sudo_support
 def get_new_command(command):
-    return u'{} --no-preserve-root'.format(command.script)
+    return '{} {} {}'.format(command.script_parts[0], '-rf', ' '.join(command.script_parts[1:]))
```

Comment:
The way it is, the rule is not doing what it is supposed to do. All these changes should be removed from the CL. Only the `side_effect` function should be part of it.